### ISSUE TITLE
Latest release (1.4.0) not working if athenaclirc file not in path

### ISSUE THREAD

**Issue by pdpark** [dbcli/athenacli#56]
athenacli works with version 1.3.3, but I get this message when trying to run athenacli 1.4.0:
```
Required argument `s3_staging_dir` or `work_group` not found.

There was an error while connecting to AWS Athena. It could be caused due to
missing/incomplete configuration. Please verify the configuration in ~/.athenacli/athenaclirc
and run athenacli again.
```

**Comment by zzl0** [comment]
Thanks for the feedback, taking a look

**Comment by zzl0** [comment]
@pdpark sorry to ask for this. But I cannot reproduce the error on my side if have set `s3_staging_dir ` either in `athenaclirc ` or `AWS_ATHENA_S3_STAGING_DIR` env var.

Do you mind providing more info to reproduce this problem? how do you set the value for s3_staging_dir?

**Comment by pdpark** [comment]
I found the issue and submitted PR #57  to fix it. A change in the 1.4.0 release requires that the `athenaclirc` file must be in the path. This was not the case before - since the default location of the file is hard-coded in `main.py` file, we know where to find it.

**Comment by zzl0** [comment]
Thanks @pdpark . I made a mistake when I was verifying it, my virtual env had the old version, but I thought it was the new version.

**Comment by pdpark** [comment]
NP, thanks for the quick response!

**Comment by zzl0** [comment]
Fixed in release v1.4.1. I also did some minor changes. Feel free to reopen this if you the issue still exists.
